### ISSUE TITLE
Auto-update script not running after installation

### ISSUE THREAD

**Issue by scandox** [evercam/evercam-gateway#27]
As part of the installation procedure we install a crontab task like so:

    sudo crontab -l | { cat; echo "*/5 * * * * /opt/evercam/evercam-gateway/bin/auto-update.sh"; } | crontab -

Sometimes it doesn't appear to be running. @rajmohanb can you try re-running this? Since you're not using sudo, just do it as root for now. I want to see if it is not working for some other reason.

**Comment by rajmohanb** [comment]
@scandox - you want me to rerun the auto-update script manually?

**Comment by scandox** [comment]
@rajmohanb - well ideally I'd like the cron script to be working. Did you
install it?

Dan

**Comment by rajmohanb** [comment]
Yes, it was installed as part of the installation. Here is the output of "crontab -l"

rajmohan@sandbox:~$ crontab -l
*/5 * * * * /opt/evercam/evercam-gateway/bin/auto-update.sh
*/5 * * * * /opt/evercam/evercam-gateway/bin/auto-update.sh
rajmohan@sandbox:~$

**Comment by scandox** [comment]
@rajmohan - the point I was trying to make is that is the crontab for user
Rajmohan that won't be able to update /opt/evercam/evercam-gateway....can
you please install it as root? Just until we have a dedicated gateway user
configured in installer. Thanks!

Dan
